deployer/auth.py

```diff
@@ -89,12 +91,31 @@ def ensure_client(self, name, domains, connection_name):
             self._ensure_client_callback(client, domains)
 
         current_connections = self.get_connections()
+
+        if connection_name == 'password':
+            # All hubs attached to a single database 'connection'
+            # will share username / password. So we create one
+            # connection per hub.
```

Comment:
When I first read this comment, my english failed me and actually understood it the other way around (that the hubs share the same database).
@yuvipanda, do you want to rephrase this a bit to make it "english-bad-day" proof? :sweat_smile:

Reply:
@GeorgianaElena done!

Reply:
Ty! :D